app/src/main/java/com/globant/pokemontcgapp/util/Data.kt

```diff
@@ -0,0 +1,9 @@
+package com.globant.pokemontcgapp.util
+
+data class Data<RequestData>(var status: Status, var data: RequestData? = null, var error: Exception? = null)
```

Comment:
put this data class and this enum in the ViewModel that use them. Do not use a generic one like this pls

Reply:
changed, thanks